src/patients/visits/VisitTab.tsx

```diff
@@ -15,26 +10,9 @@ interface Props {
 }
 
 const VisitTab = ({ patientId }: Props) => {
-  const { t } = useTranslator()
-  const { permissions } = useSelector((state: RootState) => state.user)
   const [showAddVisitModal, setShowAddVisitModal] = useState(false)
   return (
     <>
-      <div className="row">
-        <div className="col-md-12 d-flex justify-content-end">
```

Comment:
>
> Remove "add visit" button from visits tab (since there's one already) and add a "add imaging request" button instead
>
Thank @Fibii 
kindly note that this was not in the issue specification. This button is needed for UI/UX consistency with the rest of the design.